cli/lfc/src/main/java/org/lflang/cli/Lfc.java

```diff
@@ -107,16 +108,22 @@ public class Lfc extends CliBase {
       description = "Specify the runtime scheduler (if supported).")
   private String scheduler;
 
-  @Option(
-      names = {"-t", "--threading"},
-      paramLabel = "<true/false>",
-      description = "Specify whether the runtime should use multi-threading" + " (true/false).")
-  private String threading;
+  /** Mutually exclusive options related to threading. */
+  static class ThreadingMutuallyExclusive {
+    @Option(
+        names = "--single-threaded",
+        arity = "0",
+        description = "Specify whether the runtime should be single-threaded.")
```

Comment:
If the arity is zero, then this description is not correct. Should it be:
```suggestion
        description = "Specify that the runtime should be single-threaded.")
```